docs/getting-started/mainnet.md

```diff
@@ -254,229 +196,96 @@ WantedBy=multi-user.target
-  - `--min-self-delegation`: The lower threshold of validator's self-delegation amount, if the self-delegation drops below this number, all staked funds to the validator will be automatically unbonded and the validator will be inactive.
-  :::
-
-
-### Step 3-6. Check your validator status
-Once the `create-validator` transaction completes, you can check if your validator has been added to the validator set:
-
-```bash
-$ ./chain-maind tendermint show-address
-## [crocnclcons... address] ##
-$ ./chain-maind query tendermint-validator-set | grep -c [crocnclcons...]
-## 1 = Yes; 0 = Not yet added ##
-```
-
-To further check if the validator is signing blocks, kindly run this [script](https://github.com/crypto-com/chain-docs/blob/master/docs/getting-started/assets/signature_checking/check-validator-up.sh), for example:
-
-```bash
-$ curl -sSL https://raw.githubusercontent.com/crypto-org-chain/chain-docs/master/docs/getting-started/assets/signature_checking/check-validator-up.sh | bash -s -- \
---tendermint-url https://mainnet.crypto.org:26657 \
---pubkey $(cat ~/.chain-maind/config/priv_validator_key.json | jq -r '.pub_key.value')
-
-The validator is in the active validator set under the address  <YOUR_VALIDATOR_ADDRESS>
-The validator is signing @ Block#<BLOCK_HEIGHT> 👍
-```
-
-Congratulations! You've successfully set up a mainnet node and performed some basic transactions! You may refer to [Wallet Management](https://crypto.org/docs/wallets/cli.html#chain-maind) for more advanced operations and transactions.
-
-
-##  Basic Transactions and queries
-
-###  `query bank balances` - Check your transferable balance
-
-You can check your _transferable_ balance with the `balances` command under the bank module.
-:::details Example: Check your address balance
-
-```bash
-$ ./chain-maind query bank balances cro1quw5r22pxy8znjtdkgqc65atrm3x5hg6vycm5n
-
-  balances:
-  - amount: "10005471622381693"
-    denom: basecro
-  pagination:
-    next_key: null
-    total: "0"
-```
-
-:::
-
-###  `tx bank send` - Transfer operation
-
-Transfer operation involves the transfer of tokens between two addresses.
-
-#### **Send Funds** [`tx bank send <from_key_or_address> <to_address> <amount> <network_id>`]
-
-:::details Example: Send 10cro from an address to another.
-
-```bash
-$ ./chain-maind tx bank send Default cro1j7pej8kplem4wt50p4hfvndhuw5jprxxn5625q 10cro --chain-id="crypto-org-chain-mainnet-1" --gas-prices 0.1basecro
-  ## Transaction payload##
-  {"body":{"messages":[{"@type":"/cosmos.bank.v1beta1.MsgSend","from_address"....}
-confirm transaction before signing and broadcasting [y/N]: y
-```
+::: tip Remarks: 
+  - You can query the node syncing status by
+    ```bash
+    $ ./chain-maind status 2>&1 | jq '.SyncInfo.catching_up'
+    ```
+    If the above command returns `false`, It means that your node **is synced**; otherwise, it returns `true` and implies your node is still catching up.
```

Comment:
```suggestion
If the above command returns `false`, it means that your node **is synced**; otherwise, it returns `true` and implies your node is still catching up.
```